communitiesuk/funding-service-design-frontend
# FS-2390: Fix compatible html elements

### Change description

For lists in html, we should ensure if we have alternative elements they're encapsulated in a list item.  This change does that on two instances that were raised by [axe dev tools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd).  Some translations weren't working because they included % signs, I've fixed them as well while I was in the area.

### How to test

Checkout the branch, re-start your docker container to ensure the new compiled translations are picked up.  Then visit the all questions page for [cof r2w3](http://localhost:3008/all_questions/COF/R2W3?lang=cy), you can observe the fixed translations.  You can also run the axe analysis using the extension I linked above. 

### Screenshots of axe analysis

![image](https://user-images.githubusercontent.com/117724519/225591166-c1074cc0-418e-4699-a18b-d933e58d1483.png)


Note: This also included pre-commit-ci changes, which fixed issues throughout the entire codebase, hopefully we'll only have these once per repo.
